src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -44,6 +49,14 @@
  * @since 0.7
  */
 public final class ManifestsTest {
+    // @checkstyle JavadocVariableCheck (8 lines)
```

Comment:
@nesterov-n have you seen that practice in other files? Generally we have Javadocs everywhere